api/server/cluster.go

```diff
@@ -57,6 +52,22 @@ func (c *clusterApi) String() string {
 	return c.name
 }
 
+// swagger:operation GET /cluster/enumerate cluster enumerate enumerateCluster
+//
+// Lists cluster Nodes.
+//
+// This will return the entire cluster object and it's nodes.
+//
+// ---
+// produces:
+// - application/json
+// responses:
+//   '200':
+//      description: current cluster state
```

Comment:
Can we add description in the form of complete sentence as this is going to be a part of the swagger docs?

Reply:
Yes we can i just took what they used in the swagger documentation